Xamarin.Forms.Controls/GalleryPages/CollectionViewGalleries/MultiTestObservableCollectionModifier.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Xamarin.Forms.Controls.GalleryPages.CollectionViewGalleries
+{
+	internal abstract class MultiTestObservableCollectionModifier : CollectionModifier
+	{
+		protected MultiTestObservableCollectionModifier(CollectionView cv, string buttonText) : base(cv, buttonText)
+		{
+		}
+
+		protected override void OnButtonClicked()
+		{
+			if (!ParseIndexes(out int[] indexes))
+			{
+				return;
+			}
+
+			if (!(_cv.ItemsSource is MultiTestObservableCollection<CollectionViewGalleryTestItem> observableCollection))
+			{
+				return;
+			}
+
+			ModifyObservableCollection(observableCollection, indexes);
```

Comment:
I think it's better to move this call to the condition above `if (_cv.ItemsSource is ...`